frontend/src/components/@common/Button/Button.stories.tsx

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable react/jsx-key */
+import { Meta, StoryObj } from '@storybook/react';
+import { StoryContainer, StoryItemContainer, StoryItemTitle } from 'styles/storybook';
+
+import Button, { Variant, Size, Align } from './Button';
+import { HomeIcon, CalendarIcon, PlusCircleIcon, SearchIcon, StatsIcon } from 'assets/icons';
+
+const meta = {
```

Comment:
```suggestion
const meta: Meta<typeof Button> = {
```
이거 이렇게 쓰는거랑 satisfies 키워드 쓴 거랑 무슨 차이인지 궁금해요!

Reply:
저도 스토리북에서 제공하는 테스트 파일을 가져다가 쓴거라 잘 모르겠네요😅
[satisfies 공식문서](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator)를 참고해주세요ㅎㅎ